torch/csrc/jit/passes/quantization/helper.h

```diff
@@ -89,7 +95,37 @@ findChildModule(const Module& module, const std::vector<std::string>& path);
 // Given an CallMethod node, get the module instance corresponding
 // to the instance Value
 TORCH_API Module getInvokedModule(Module& module, Node* n, Value* self);
-// =========== helper functions for Module  ==========
+
+// ==================== filter functions for matches ==============
+auto aten_add_alpha_is_one =
+    [](const Match& match,
+       const std::unordered_map<std::string, Value*>& vmap) {
+      const auto& match_vmap = match.values_map;
+      auto alpha = toIValue(match_vmap.at(vmap.at("alpha")));
+      return alpha && alpha->isInt() && alpha->toInt() == 1;
+    };
+
+auto is_functional_relu =
+    [](const Match& match,
+       const std::unordered_map<std::string, Value*>& vmap) {
+      const auto& match_vmap = match.values_map;
+      Value* relu = match_vmap.at(vmap.at("relu"));
+      return relu->type()->cast<FunctionType>() && getFuncName(relu) == "relu";
+    };
+
+auto is_relu_module = [](const Match& match,
```

Comment:
Can we make these functions general, so that they take a generic function name, for example:
``` c++
auto is_functional =
    [](const Match& match,
       const std::unordered_map<std::string, Value*>& vmap,  std::string functional_name) {
      const auto& match_vmap = match.values_map;
      Value* relu = match_vmap.at(vmap.at(functional_name));
      return relu->type()->cast<FunctionType>() && getFuncName(relu) == functional_name;
    };
```